### PR TITLE
[DOC-5778] DRApps README review

### PR DESCRIPTION
**Pull request by n-aylward**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

A copyedit of the README for the DRApps directory.

### CODE REVIEW

File: drapps/README.md [hunk 1]
```diff
@@ -1,110 +1,100 @@
...
 
-Path to folder which will be used to create custom application. Files from this folder
-will be uploaded to Data Robot and used for creating custom application image. Custom
-application will be started from this image.
+To test this, deploy an example Streamlit app using the following command from 
+the [`./drapps`](https://github.com/datarobot/datarobot-user-models/tree/master/drapps) directory:
 
-To use the current working directory use `--path .`.
+``` sh
+./drapps.py -t <your_api_token> -e "[Experimental] Python 3.9 Streamlit" -p ./demo-streamlit
```
Comment (n-aylward) [inline drapps/README.md:57]: For the `demo-streamlit` directory, I would also recommend fixing the typo in the filename of `demo-streamlet.py` (and the associated command in `start-app.sh`).

Comment (demchukilya) [reply drapps/README.md:57]: Oh, thank you.

**Review by demchukilya — APPROVED** [review]
LGTM

**Review by yakov-g — APPROVED** [review]

File: drapps/README.md (hunk 1 above)
Comment (n-aylward) [reply drapps/README.md:57]: @demchukilya Also not sure if the `[Experimental]` tag  on the environment will still be there when the feature is GA, but I didn't want to make that kind of change in this review.

Comment (demchukilya) [reply drapps/README.md:57]: I think we should keep Experimental at least for now.

Comment (n-aylward) [reply drapps/README.md:57]: Sounds good! Feel free to merge this whenever! Not sure if we need more reviews.

**Review by chrj — APPROVED** [review]

**Pull request merged**